app/settings.py

```diff
@@ -1,8 +1,33 @@
 import os
+from pathlib import Path
+
+BASE_DIR = Path(__file__).resolve(strict=True).parent.parent
+ENV_FILE = os.getenv('ENV_FILE', f'{BASE_DIR}/.env.local')
+if ENV_FILE and Path(ENV_FILE).exists():
+    from dotenv import load_dotenv
+
+    print(f"Running locally hence injecting env vars from {ENV_FILE}")
```

Comment:
We should not use print statement but here it is fine as it is only for local development, this line should not be reached on k8s deployment.